validator/src/commands/contact_info/mod.rs

```diff
@@ -36,3 +50,43 @@ pub fn execute(matches: &ArgMatches, ledger_path: &Path) {
         print!("{contact_info}");
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use {
+        super::*,
+        crate::commands::tests::{
+            verify_args_struct_by_command, verify_args_struct_by_command_is_error,
+        },
+    };
+
+    #[test]
+    fn verify_args_struct_by_command_contact_info_output_json() {
+        verify_args_struct_by_command(
+            command(&DefaultArgs::default()),
+            vec!["contact-info", "--output", "json"],
```

Comment:
Some of these repeated strings could get annoying. We could add a non-public constant:
```rust
CONST COMMAND: &str = "contact-info";
```
This might be overkill tho, and doing it for each arg also might be overkill; thoughts ?

Reply:
sounds good! 4ab4e4a3b7